### ISSUE TITLE
Ruby 2.4.0 causing hash error in tests

### ISSUE THREAD

**Issue by sublimecoder** [vcr/vcr#637]
So I managed to narrow down that VCR is causing multiple test failures in my suite while trying to upgrade to ruby 2.4.0

The error I see is this.
``` bash
undefined method `<<' for {:read_timeout=>60, :continue_timeout=>nil, :debug_output=>nil}:Hash
```

I'm not exactly certain where inside of VCR it comes from but I do know its exclusive to 2.4.0, which I assume has something to do with the 2.4.0 hash changes.

Has anyone worked out a fix for this or is there a fix in progress?

**Comment by mcfiredrill** [comment]
Thanks for reporting this!  2.4 support is currently a WIP, there is a WIP PR open. 
I hoped to work on it myself when I had time but of course contributions from anyone else are welcome. :pizza: 

iPhoneから送信

2017/02/17 6:34、Jared Smith <notifications@github.com> のメッセージ:

**Comment by Onumis** [comment]
not sure if 100% the same, but I had an identical error and updating webmock fixed it:
https://github.com/bblimke/webmock/blob/master/CHANGELOG.md#231

**Comment by itskingori** [comment]
I believe this issue can be closed ... @Onumis [solution][1] worked.

[1]: https://github.com/vcr/vcr/issues/637#issuecomment-285439818

**Comment by AmitPatel-BoTreeConsulting** [comment]
I got the error again with `vcr (3.0.3)` `Ruby 2.4.1p111` and `rails (5.1.3)`

**Comment by krisleech** [comment]
I get the error with vcr 3.0.3 and fakeweb 1.3.0

There is a fix for fakeweb, which adds Ruby 2.4 compatibility and fixes this error here: https://github.com/chrisk/fakeweb/commit/5bd70b72962081e46c7b22a39d00b7660d0ad30e

This is unrelased and on master branch, you could point to master branch until the next release.

Or it is "fixed" by changing from fakeweb or webmock.
